lib/tests/testscripts/scripts/reporting/multisystem-evaluate-measure.xml

```diff
@@ -0,0 +1,276 @@
+      </operation>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the returned HTTP status is 200(OK)"/>
+        <operator value="in"/>
+        <responseCode value="200"/>
+      </assert>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the returned format is XML. Warning only as the server may not support the return of response content."/>
+        <contentType value="xml"/>
+        <warningOnly value="true"/>
+      </assert>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the return type is MeasureReport. Warning only as the server may not support the return of response content."/>
+        <resource value="MeasureReport"/>
+        <warningOnly value="true"/>
+      </assert>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the returned HTTP Header Last-Modified is present. Warning only as the server may not support versioning."/>
+        <headerField value="Last-Modified"/>
+        <operator value="notEmpty"/>
+        <warningOnly value="true"/>
+      </assert>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the returned HTTP Header ETag is present. Warning only as the server may not support versioning."/>
+        <headerField value="ETag"/>
+        <operator value="notEmpty"/>
+        <warningOnly value="true"/>
+      </assert>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the returned HTTP Header Location is present. Warning only as this is optional but servers are encouraged to return this."/>
+        <headerField value="Location"/>
+        <operator value="notEmpty"/>
+        <warningOnly value="true"/>
+      </assert>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the returned resource type is MeasureReport."/>
+        <resource value="MeasureReport"/>
+      </assert>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the returned MeasureReport conforms to the base FHIR specification."/>
+        <validateProfileId value="measurereport-profile"/>
+        <warningOnly value="true"/>
+      </assert>
+    </action>
+    <action>
+      <assert>
+        <description value="Confirm that the returned MeasureReport contains the expected initial population."/>
+        <operator value="equals"/>
+        <compareToSourceId value="dest1-measurereport"/>
+        <compareToSourcePath value="//fhir:MeasureReport/fhir:group/fhir:population[fhir:code[fhir:coding[fhir:code[@value='initial-population']]]]/fhir:count/@value"/>
+        <path value="//fhir:MeasureReport/fhir:group/fhir:population[fhir:code[fhir:coding[fhir:code[@value='initial-population']]]]/fhir:count/@value"/>
+      </assert>
```

Comment:
This assertion fails for me. If you look at the [results on kanvix](http://measure.eval.kanvix.com/cqf-ruler/baseDstu3/Measure/measure-col/$evaluate-measure?patient=Patient-6516&periodStart=1991-01-01&periodEnd=1991-12-31), they're not falling into the IPP, and they do on my local instance. Please confirm I've input the proper base URL.

Reply:
I believe this got resolved as a result of me properly creating the resources before doing the evaluation